lib/qx/tool/cli/commands/Lint.js

```diff
@@ -105,7 +113,11 @@ qx.Class.define("qx.tool.cli.commands.Lint", {
           let outputFormat = this.argv.format || "codeframe";
           const formatter = linter.getFormatter(outputFormat);
           const s = formatter(report.results);
-          if (report.errorCount > 0 || this.argv.warnAsError) {
+          if (this.argv.outputFile) {
+            console.info(this.argv.outputFile);
```

Comment:
Is this `console.info()` left from debugging or intentional to inform the user? If the latter, please add a `-v`/`--verbose` CLI option (you can copy/paste from other commands) and wrap the output into `if (this.argv.verbose` {}`.